src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1048,6 +1086,17 @@ int sd_dhcp_server_forcerenew(sd_dhcp_server *server) {
         return r;
 }
 
+int sd_dhcp_server_set_bind_to_interface(sd_dhcp_server *server, int enabled) {
+        assert_return(server, -EINVAL);
```

Comment:
Maybe, we should refuse this when the server is already started.

Reply:
Hi! Right, the way as is it will just break the server. I added a check for this.